packages/aws-cdk-lib/aws-rds/lib/cluster-engine.ts

```diff
@@ -895,10 +944,7 @@ export class AuroraPostgresEngineVersion {
   public static readonly VER_13_13 = AuroraPostgresEngineVersion.of('13.13', '13', { s3Import: true, s3Export: true });
   /** Version "14.3". */
   public static readonly VER_14_3 = AuroraPostgresEngineVersion.of('14.3', '14', { s3Import: true, s3Export: true });
-  /**
-   *  Version "14.4".
-   * @deprecated Version 14.4 is no longer supported by Amazon RDS.
-   */
+  /**  Version "14.4". */
```

Comment:
Was this wrongly deprecated previously?

Reply:
I would assume so, unless it was deprecated then reinstated?

```sh
$ aws rds describe-db-engine-versions --engine aurora-postgresql --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion=="14.3") | .Status'
available

$ aws rds describe-db-engine-versions --engine aurora-postgresql --region eu-west-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion=="14.3") | .Status'
available
```